### PR TITLE
Document how to use Docker Compose integration when running tests

### PR DESCRIPTION
**Pull request by OnSuorce**
Added note to indicate how to enable `spring-boot-docker-compose` module in `@SpringBootTest` tests

### CODE REVIEW

**Comment by OnSuorce** [comment]
Hi,
Did i write the pull request message wrongly? It seems to me that the issue #35661  was not recognized.

**Comment by wilkinsona** [comment]
No, everything's fine. I don't think GitHub lets you reference an issue from the title of an issue or PR, that's all. Thanks for the PR.

File: spring-boot-project/spring-boot-docs/src/docs/asciidoc/features/docker-compose.adoc [hunk 1]
```diff
@@ -37,6 +37,7 @@ When this module is included as a dependency Spring Boot will do the following:
 
 NOTE: The `docker compose` or `docker-compose` CLI application needs to be on your path in order for Spring Boot’s support to work correctly.
 
+NOTE: If you wish to run docker compose via `spring-boot-docker-compose` when using `@SpringBootTest` 's tests, the property `spring.docker.compose.skip.in-tests` must be set to `false`
```
Comment (wilkinsona) [inline spring-boot-project/spring-boot-docs/src/docs/asciidoc/features/docker-compose.adoc:40]: Can you please replace `` `spring.docker.compose.skip.in-tests` `` with `configprop:spring.docker.compose.skip.in-tests[]`. `configprop` is a macro that runs as part of building the documentation. It verifies that the configuration property exists, helping to ensure the accuracy of the documentation.

**Review by wilkinsona — CHANGES_REQUESTED** [review]

**Comment by OnSuorce** [comment]
> No, everything's fine. I don't think GitHub lets you reference an issue from the title of an issue or PR, that's all. Thanks for the PR.

Ok, next time i'll reference it from the description

**Comment by wilkinsona** [comment]
@OnSuorce thank you very much for making your first contribution to Spring Boot.

**Comment by OnSuorce** [comment]
@wilkinsona Thank you for your help. Looking forward to contributing again

**Pull request closed without merging**